kahuna/public/js/search/results.js

```diff
@@ -154,6 +154,9 @@ results.controller('SearchResultsCtrl', [
             }
 
             return images;
+        }).catch(error => {
+            ctrl.loadingError = error;
```

Comment:
If the user doesn't have a session and gets a 401 back, or if the server is broken or down (5xx).